.ci/configure_tracing_for_kata.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
+
+[ "$#" -eq 1 ] || die "Specify enable or disable"
+
+default_kata_cfg_file="/usr/share/defaults/kata-containers/configuration.toml"
```

Comment:
Or you could use the `$RUNTIME_CONFIG` type elements, but probably have to call an init func from the lib.sh to get those setup, or you could use `kata-runtime kata-env --json | jq...` to extract the actual path (which is what the lib.sh does, but hopefully now uses the actual runtime configured in the docker setup these days, so in theory should pick up the correct runtime and thus the correct config file).

Reply:
agree, changed to use `kata-env`